doc/src/pair_sw.rst

```diff
@@ -14,7 +14,15 @@ Syntax
 
 .. code-block:: LAMMPS
 
-   pair_style sw
+   pair_style style keywords values
+
+* style = *sw* or *sw/gpu* or *sw/omp*
```

Comment:
We usually don't list individual accelerated pair styles like this. See, e.g. the `pair_airebo.rst` documentation file, which documents multiple variants of AIREBO *and* accelerated variants.

Reply:
Removed these changes.